arch/arm/src/Makefile

```diff
@@ -190,7 +190,7 @@ EXTRA_LIBS += allsyms$(OBJEXT)
 endif
 
 define LINK_ALLSYMS_KASAN
-	$(if $(CONFIG_ALLSYMS),
+	$(if $(CONFIG_ALLSYMS),)
```

Comment:
but it's the right if make expression

Reply:
Normally it is "$(if $(CONFIG_ALLSYMS), something, etc)", the "$(if" normally needs a closing ")"

Reply:
no, please watch ci or change your change locally.

Reply:
Oh ok, I saw the issue! I'll close it.